src/lib/inputs/InputRadio/input-radio-documentation/input-radio.examples.tsx

```diff
@@ -12,8 +13,42 @@ export const inputRadioExample1: React.FC<any> = () => (
     {formikProps => (
       <>
         Value: {formikProps.values.sampleRadio}
-        <InputRadio id="sampleRadio.yes" label="Yes" name="sampleRadio" value="Yes" formikProps={formikProps} />
-        <InputRadio id="sampleRadio.no" label="No" name="sampleRadio" value="No" formikProps={formikProps} />
+        <InputRadioGroup
+          formikProps={formikProps}
+          legend="Sample Radio"
+          name="sampleRadio"
+          value={formikProps.values.sampleRadio}
+        >
+          <InputRadio id="sampleRadio.yes" label="Yes" value="Yes" />
+          <InputRadio id="sampleRadio.no" label="No" value="No" />
+        </InputRadioGroup>
+      </>
+    )}
+  </Form>
+);
+
+export const inputRadioExample2: React.FC<any> = () => (
+  <Form
+    initialValues={{
+      sampleRadio: 'Yes'
+    }}
+    onSubmit={() => undefined}
+  >
+    {formikProps => (
+      <>
+        Radio buttons aligned horizontally
+        <br />
+        Value: {formikProps.values.sampleRadio}
+        <InputRadioGroup
+          formikProps={formikProps}
+          legend="Sample Radio"
+          name="sampleRadio"
+          row
+          value={formikProps.values.sampleRadio}
```

Comment:
same with this value (can be assumed from the name and formikProps)

Reply:
Same here, this is resolved too.